covalent/_shared_files/config.py

```diff
@@ -209,7 +217,7 @@ def set(self, key: str, value: Any) -> None:
         Returns:
             None
         """
-        set_key(find_dotenv(CONFIG_FILE_NAME), key, str(value))
+        set_key(self.config_file, key, str(value))
```

Comment:
@AlejandroEsquivel I think this was the main offending line

Reply:
yep it was, it was a bit obscure what find_dotenv was doing under the hood